WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignCategoryThumbnailSize.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+enum SiteDesignCategoryThumbnailSize {
+    case category
+    case recommended
+
+    var value: CGSize {
+        switch self {
+        case .category where UIDevice.isPad():
+            return .init(width: 250, height: 325)
+        case .category:
+            return .init(width: 200, height: 260)
+        case .recommended where UIDevice.isPad():
+            return .init(width: 327, height: 450)
```

Comment:
I know it comes from the design specs, but 327 seems kind of a weird number here. Not a blocker for this PR at all, but @kyleaparker do you think we should round it up? Also it feels a bit odd that the recommended width on iPhone is larger than on iPad; I realize this is to obtain the effect of having one prominent recommended design on iPhone and two on iPad, but this might not be always the case depending on the actual screen size.

Reply:
Yea the 327 comes from fitting exactly two designs within the space but I'm assuming it won't be possible to dynamically adjust the sizes? We could round to 300 to clean up the numbers 👍

Reply:
👋 @Gio2018 and @kyleaparker I'm going to go ahead and merge this PR into the feature branch with the current values. If we need to tweak them further we'll just make a new PR. Thanks!

Reply:
> but I'm assuming it won't be possible to dynamically adjust the sizes? 

we can adjust sizes. If we do so, we likely need to define how the sizes should behave when the screen width is larger than the height (landscape). In that case, I assume we probably would not want to fit them horizontally.
Numbers don't need to be rounded, it just seemed a bit unusual.